.gitignore

```diff
@@ -4,3 +4,4 @@
 .idea*/**
 /bin/
 boots
+ipxe/bin/ipxe.efi
```

Comment:
This is covered by `ipxe/.gitignore` and should be removed here. Looking forward to this landing!